src/chat/chat.controller.ts

```diff
@@ -89,8 +88,8 @@ export class ChatController {
   }
 
   // Delete a Chat
-  @Delete('/delete')
-  async deleteChat(@Res() res, @Query('uid') uid) {
+  @Delete('/:uid')
+  async deleteChat(@Res() res, @Param('uid') uid: string) {
```

Comment:
Should delete have only uid?
Like when we delete chat we delete some message and one uid can have many message.

This will delete whole of the chat, again that can be the use case, so please create a new controller for deleteing some specific chat by id

Reply:
Yeah, . will do that  , thanks